packages/react-core/src/evm/hooks/async/marketplace.ts

```diff
@@ -1380,6 +1381,71 @@ export function useBuyNow(contract: RequiredParam<Marketplace>) {
   );
 }
 
+/**
+ * Buy a direct listing from a MarketplaceV3 contract
+ *
+ * @example
+ * ```jsx
+ * const Component = () => {
+ *   const { contract } = useContract("{{contract_address}}", "marketplace-v3");
+ *   const {
+ *     mutate: buyNow,
+ *     isLoading,
+ *     error,
+ *   } = useBuyFromListing(contract);
+ *
+ *   if (error) {
+ *     console.error("failed to buy direct listing", error);
+ *   }
+ *
+ *   return (
+ *     <button
+ *       disabled={isLoading}
+ *       onClick={() => buyNow({listingId: 1, quantity: 1, buyer: "{{address}}"})}
+ *     >
+ *       Buy listing!
+ *     </button>
+ *   );
+ * };
+ * ```
+ *
+ * @param contract - an instance of a MarketplaceV3 contract
+ * @returns a mutation object that can be used to buy out a direct listing
+ * @see {@link https://portal.thirdweb.com/react/react.useBuyFromListing?utm_source=sdk | Documentation}
+ * @beta
+ */
+export function useBuyFromListing(contract: RequiredParam<MarketplaceV3>) {
```

Comment:
```suggestion
export function useBuyDirectListing(contract: RequiredParam<MarketplaceV3>) {
```

this name makes more sense to me, goes more in line with the other marketplace v3 hooks

Reply:
Done, thanks for the suggestions